tests/e2e/playback/controls/player/radio-button-widget.e2e.js

```diff
@@ -0,0 +1,94 @@
+import { Selector } from "testcafe";
+
+const path = require("path");
+
+const radio_selected = Selector(".custom-control-input");
+const radio_label = Selector(".custom-control-label");
+
+import { testcafe_page_visual_regression } from "@curi-bio/frontend-test-utils";
+
+fixture`playback/controls/player/radiobutton-widget/basic-radio`
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/radiobutton-widget/basic-radio`; // specify the start page
+
+test("testing the Radio Button Widget for the NO pre-selected option Entered", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "radio-widget"
+  );
+  const screenshot_path = path.join(
+    screenshot_path_base,
+    "basic-radio-no-selection"
+  );
+  await testcafe_page_visual_regression(t, screenshot_path);
+});
+
+test("testing the Radio Button Widget for the NO pre-selected option Entered a Hover Effect", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "radio-widget"
+  );
+  const screenshot_path = path.join(
+    screenshot_path_base,
+    "basic-radio-button-hover"
+  );
+  await t.hover(radio_label.nth(0));
+  await testcafe_page_visual_regression(t, screenshot_path);
+});
+
+fixture`playback/controls/player/radiobutton-widget/basic-radio-preselect`
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/radiobutton-widget/basic-radio-preselect`; // specify the start page
+
+test("testing the Radio Button Widget for the ON selected option Entered", async (t) => {
```

Comment:
it is unclear to me what you are testing here.  If your page already begins with radio index 1 being pre-selected, then clicking on that button should not have any effect. are you trying to test whether the pre-select works? or whether clicking on radio button index 1 causes that button to be selected?  in the current test it seems like you could use the `basic-radio` page and the image result would still be the same

The Jest ESLint plugin does not work well on Testcafe files, but we still want to use the same Given/When/Then format to describe what we are trying to test so that it is clear to someone reading the code what is being tested, and it is clear while writing the code what the goal of the test is so that you can confirm that the test is doing what you hope it will

Reply:
`Input[type='radio']` has the following set of states
a) checked (one among the group) or (none of the radio's checked)
b) focus (one among the group which is checked)
c) after and before

Unlike other items of just focus/blur states of other dom elements, for now I have introduced a pre-select with no focus and pre-select with a click which results in focus and the event getting trigger providing (index : <num> , name : <label text>)

Added an additional VRT with the value of Label and Index on a div.